### PR TITLE
Create the page for sensitive content explanation

### LINKED ISSUE #2550: Add a page explaining the sensitive content filtering approaches
## Problem

The content safety wall #2462 shows [different text based on different values](https://github.com/WordPress/openverse/issues/791#issuecomment-1484245797) of the `sensitivity` field provided by the API #2549.

## Description

These differences in text can be subtle and may not immediately make clear to the user why a particular media has been flagged as sensitive. It would be helpful to have a page explaining the different factors considering when flagging a media item.

Essentially it would be mapping of keys in the `sensitivity` field in the API response to explanatory text that can be linked to from the content safety wall if the user needs more information about our process.

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Fixes #2550 by @dhruvkb

## Description

This PR adds the page for explanation about sensitive content and our filtering techniques.

## Testing Instructions

0. Check out the PR and run the dev server.
1. Visit the page directly at `/sensitive-content`.
2. Go to `/preferences` and turn on `sensitive_content` and `fake_sensitive` flags.
3. Make a search query.
4. Find a blurred result, click to open it.
5. Click the link to "Learn more". You should be taken to the `/sensitive-content` page again.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/src/pages/sensitive-content.vue [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      >
+    </i18n>
+
+    <p>{{ $t("sensitive.sensitivity.what", { openverse: "Openverse" }) }}</p>
+    <p>{{ $t("sensitive.sensitivity.how", { openverse: "Openverse" }) }}</p>
+
+    <!-- TODO
+    <h2 id="on-off">{{ $t("sensitive.onOff.title") }}</h2>
+    <p>{{ $t("sensitive.onOff.description", { openverse: "Openverse" }) }}</p>
+    -->
```
Comment (AetherUnbound) [inline frontend/src/pages/sensitive-content.vue:29]: Should this still be TODO?

**Review by AetherUnbound — APPROVED** [review]
The content looks good! I have one note about a TODO, and also, I noticed that the "sensitive" link in the Safe Browsing section of the search results page links to `/about`, should that link to `/sensitive-content` now?
Screenshot_2023-09-29_17-08-56

File: frontend/src/pages/sensitive-content.vue (hunk 1 above)
Comment (dhruvkb) [reply frontend/src/pages/sensitive-content.vue:29]: I kept this as TODO as this is a TODO in the reference Google doc. Alternatively I can write it directly in the PR and we can iterate on it here?

**Comment by dhruvkb** [comment]
Good catch @AetherUnbound, thank you! Updated the link in e4be495.

**Review by obulat — CHANGES_REQUESTED** [review]
The page works well! 
The request for change is to change the title of the page from "about" to "sensitive content": the component's `name` property and the `title` in `useMeta`.

File: frontend/src/pages/sensitive-content.vue [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+<script lang="ts">
+import { defineComponent, useMeta } from "@nuxtjs/composition-api"
+
+import { useI18n } from "~/composables/use-i18n"
+
+import VLink from "~/components/VLink.vue"
+import VContentPage from "~/components/VContentPage.vue"
+
+export default defineComponent({
+  name: "AboutPage",
```
[the remark this answers is not in the document]
Comment (dhruvkb) [reply frontend/src/pages/sensitive-content.vue]: Good catch, thanks @obulat!

File: frontend/src/pages/sensitive-content.vue [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+export default defineComponent({
+  name: "SensitiveContent",
+  components: { VLink, VContentPage },
+  layout: "content-layout",
+  setup() {
+    const i18n = useI18n()
+
+    useMeta({
+      title: `${i18n.t("about.title", {
```
Comment (obulat) [inline frontend/src/pages/sensitive-content.vue:143]: @dhruvkb, I think this title should also be changed.

**Review by obulat — APPROVED** [review]

File: frontend/src/pages/sensitive-content.vue (hunk 3 above)
Comment (obulat) [reply frontend/src/pages/sensitive-content.vue:143]: Approving to unblock

**Pull request merged**